src/components/scenes/CoinRankingScene.tsx

```diff
@@ -81,8 +81,9 @@ const CoinRankingComponent = (props: Props) => {
     const key = `${index}-${item}-${rank}-${currencyCode}-${lastUsedFiat}`
     debugLog(LOG_COINRANK, `renderItem ${key.toString()}`)
 
+    const disable = index >= MAX_LIST_ITEMS_ANIM ? 'view' : undefined
```

Comment:
If `disable` we're just a boolean this would simply be `const disable = index >= MAX_LIST_ITEMS_ANIM`

Reply:
As mentioned above. There are necessarily two types of view disables so that we don't render extra Views if not needed.